app/models.py

```diff
@@ -112,6 +112,19 @@ def delete(self):
 		db.session.delete(self)
 		db.session.commit()
 
+	def to_json(self):
```

Comment:
why did you choose to convert the event to son in this file?

Reply:
its easier to call a model method on `events/views.py` than implement conversion on every route, just need to call `event.to_json()`

Reply:
Alright